examples/02_fit_predict_plot_employee_salaries.py

```diff
@@ -73,7 +73,7 @@
     'one-hot': OneHotEncoder(handle_unknown='ignore', sparse=False),
     'similarity': SimilarityEncoder(similarity='ngram'),
     'target': TargetEncoder(handle_unknown='ignore'),
-    'minhash': MinHashEncoder(n_components=10, ngram_range=(2, 4),
+    'minhash': MinHashEncoder(n_components=100, ngram_range=(2, 4),
```

Comment:
```suggestion
    'minhash': MinHashEncoder(n_components=100,
```
If I am not wrong, this is the default parameter, so we can remove it (as well as the ones after), with the goal to make the example simpler.